### PR TITLE
Themes | Business Dashboard - Misc Sidebar Fixes

### PR DESCRIPTION
**Pull request by ronenst**
### 🔦 Summary
Fixes in Sidebar for Business Dashboard Theme
* Fix dark skin settings leaking into light skin (undefined for theme) [link](https://jira.wixpress.com/browse/WOSBM-3580?focusedCommentId=8500187&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-8500187)
* Various fixes to `SidebarSectionItem` stylings [link](https://jira.wixpress.com/browse/WOSBM-3580?focusedCommentId=8500187&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-8500187)

### ✅ Checklist

- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 🔬 Change is tested
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6295

File: src/SidebarBackButton/test/SidebarBackButton.visual.js [hunk 1]
```diff
@@ -25,7 +30,22 @@ export const runTests = (
         }`,
         module,
       ).add(it, () =>
-        testWithTheme(<SidebarBackButton>Go Back</SidebarBackButton>),
+        testWithTheme(
```
Comment (EgoziE) [inline src/SidebarBackButton/test/SidebarBackButton.visual.js:33]: 1. We don't recommend changing visual tests in the same PR with code changes, this way you might break something without noticing, I suggest adding visuals first (we'll merge it quickly).
2. Please add a gray background color (#f0f4f7 is fine) to the "light skin" test in order to test it correctly.

image

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/SidebarBackButton/test/SidebarBackButton.visual.js (hunk 1 above)
Comment (ronenst) [reply src/SidebarBackButton/test/SidebarBackButton.visual.js]: Thanks, opened here:
https://github.com/wix/wix-style-react/pull/6298

Please note this PR will "break" the added Dashboard Theme visual tests light skin, due to the fix here: [`10c4e4d` (#6295)](https://github.com/wix/wix-style-react/pull/6295/commits/10c4e4db952282105bf67c8c666adf54d16c4286#diff-8ae4d1d7b9c6fdf29d5d1906a4ed36b662221cc37a70db497606d68f4dea730fR15)

**Comment by ronenst** [comment]
There is also a bug with the typography of MadeFor in the sidebar skin, it should be 500 with bold being 700 but it's currently all set to 400. I understand that's a separate fix and shouldn't be handled by the sidebar styles according to this discussion:
https://wix.slack.com/archives/C01B85QAU2E/p1606034777006400

**Comment by EgoziE** [comment]
@ronenst We're working on Madefor adjustments don't worry.
Maybe you can help us merge this: https://github.com/wix-private/business-manager/pull/3002 😉

**Comment by ronenst** [comment]
@EgoziE All failing visual test seem to be bug fixes and should be approved, shall I approve them?

**Comment by EgoziE** [comment]
Leave approving to me, this is my best indicator in review :)

**Review by EgoziE — APPROVED** [review]

**Pull request merged**